kpi/tests/test_cleanup_synchronous_exports.py

```diff
@@ -0,0 +1,162 @@
+import threading
+from datetime import timedelta
+
+from constance.test import override_config
+from django.test import TransactionTestCase
+from django.db import DatabaseError, transaction
+from django.utils import timezone
+
+from kobo.apps.kobo_auth.shortcuts import User
+from kpi.constants import ASSET_TYPE_SURVEY
+from kpi.models import Asset, AssetExportSettings, SubmissionSynchronousExport
+from kpi.tasks import cleanup_synchronous_exports
+
+
+class SynchronousExportCleanupTestCase(TransactionTestCase):
+    def setUp(self):
+        self.user = User.objects.create(username='testuser')
+
+    @override_config(EXPORT_CLEANUP_GRACE_PERIOD=30)
+    def test_synchronous_exports_older_than_30_minutes_are_deleted(self):
+        # Export older than 30 min - should be deleted
+        old_export = self._create_synchronous_export(
+            minutes_old=31, asset=self._create_asset('Old Asset')
```

Comment:
You should use `EXPORT_CLEANUP_GRACE_PERIOD` + 1 and `EXPORT_CLEANUP_GRACE_PERIOD` - 1, to make it less hard coded